tests/CMakeLists.txt

```diff
@@ -287,3 +287,32 @@ ENDIF()
 # Set up the default attest configuration file:
 CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/tests/attest.conf.in
   ${CMAKE_BINARY_DIR}/attest.conf)
+
+# For the time being, run `attest` as if it were a single test.
+set(disabled_tests)
+list(APPEND disabled_tests "mpirun=[5-9]")
+list(APPEND disabled_tests "explicit_ex1_2d.mpirun=4.input")
+list(APPEND disabled_tests "explicit_ex2_3d.nodal_quadrature.input")
+list(APPEND disabled_tests "explicit_ex4_3d.mpirun=4.input")
+list(APPEND disabled_tests "explicit_ex5_3d.mpirun=2.input")
+list(APPEND disabled_tests "explicit_ex8_2d.input")
+list(APPEND disabled_tests "explicit_ex8_2d.scratch_hier.input")
+list(APPEND disabled_tests "free_falling_cyl_cib|cib_plate")
+list(APPEND disabled_tests "nwt_cylinder|rotating_barge|cib_double_shell")
+list(APPEND disabled_tests "restart")
```

Comment:
I think the restart tests are fine (as long as they don't use more than 2 cores) but we can worry about this later.

Reply:
The restart tests are fine - we just need the MPI limit.